lib/rust/ensogl/component/text/src/typeface/glsl/glyph.glsl

```diff
@@ -10,11 +14,18 @@ highp vec2 get_scaled_uv() {
 }
 
 highp vec2 get_texture_coord() {
-    highp vec2 msdf_fragment_size = input_msdf_size / vec2(textureSize(input_atlas,0));
+    highp vec2 msdf_fragment_size = input_msdf_size / vec2(textureSize(input_atlas, 0));
     highp vec2 offset             = vec2(0.0, input_atlas_index) * msdf_fragment_size;
     return offset + get_scaled_uv() * msdf_fragment_size;
 }
 
+highp float get_fatting() {
+    bool glyph_is_bold            = (input_style & STYLE_BOLD_FLAG) != 0;
+    highp vec2  local_to_px_ratio = 1.0 / fwidth(input_local.xy);
+    highp float font_size_px      = input_font_size * (local_to_px_ratio.x + local_to_px_ratio.y) / 2.0;
+    return glyph_is_bold ? font_size_px * BOLD_FATTING : 0.0;
```

Comment:
Instead of bold / not bold the user might set how much bold the font sohuld be – this will be even a nicer setting – new fonts are made this way (see dynamic bold fonts on google fonts). What do you think of such a design instead? I'm OK with the current one though!

Reply:
The questin is if we want to use Bold-version of fonts in the future. If yes, then I would keep consistent API.